pipeline/price_extraction/controller.py

```diff
@@ -0,0 +1,60 @@
+import requests
```

Comment:
Thanks for this piece of code!! It was important to automatise the price extraction too